### DISCUSSION TITLE
Links to old versions of Spring Boot 2.* documentation don't work

### DISCUSSION THREAD

**Discussion by mpesa1975** [community/community#46133] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I hope this is the correct place for the report.

Links to the old version of Spring Boot 2.* don't work on page https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-Older-Release-Notes
For example https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes
displays:
Looks like something went wrong!
We track these errors automatically, but if the problem persists feel free to contact us. In the meantime, try refreshing.
[Contact Support](https://support.github.com/contact) — [GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

See also: https://github.com/spring-projects/spring-boot/issues/34043

**Comment by paulseward** [comment]
We're seeing the same 500 errors on our private project wiki pages.  Some pages load, some don't.

If I clone the git repo containing the wiki pages (via the `github.com:/$org/$project.wiki.git` style links) all the pages are still there, but trying to access some of them throws a 500 error.

This looks like a general wiki issue?
